### DISCUSSION TITLE
How to hide Library in whiteboard?

### DISCUSSION THREAD

**Discussion by yasmagic** [mynaparrot/plugNmeet-server#607] [General]
Is there a possibility to hide Library on the whiteboard?

![image](https://github.com/user-attachments/assets/25508b8d-c143-46e5-8b04-f25bf53c1e43)

**Comment by yasmagic** [comment]
Any idea how to hide the Library tool?

**Comment by Saber-ebr** [comment]
You can remove the preload Libraries in this section of config.js file:
window.WHITEBOARD_PRELOADED_LIBRARY_ITEMS = [];

Don't know about the key itself, it's provided by the whiteboard (Excalidraw) itself.
